packages/core/src/preview.ts

```diff
@@ -1,7 +1,12 @@
 import { yellow, green, bold } from 'chalk'
-import { CREATE3_PROXY_INITCODE } from '@sphinx-labs/contracts'
```

Comment:
The preview should display something like this, right?
```
GnosisSafe<0x...>.call{ value: 0.1 ether }()
```

Did you decide not to include a preview element that says something like "Check balance of the Gnosis Safe"?

Reply:
Yeah, we only include the call to transfer funds to the Safe in the preview.